applications/DropletDynamicsApplication/python_scripts/droplet_dynamics_analysis.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+from sys import argv
+from importlib import import_module
+
+import KratosMultiphysics
+
+from KratosMultiphysics.analysis_stage import AnalysisStage
+
+class DropletDynamicsAnalysis(AnalysisStage):
```

Comment:
Doesn't FluidDynamicsAnalysis rely on a wrapper for multiple solvers? I think direct derivation from the AnalysisStage makes more sense for maximum simplicity.

Reply:
That's precisely my point, if you override the creation of the solver you can use your custom droplets dynamics wrapper. I think it would be much simpler in that way.

Reply:
Done.